helm/values.yaml

```diff
@@ -2,11 +2,11 @@ environment: development
 postgresUsername: postgres@mine-support2
```

Comment:
Postgres username still includes `mine-support`.

Reply:
This is the server name in Azure.  Can't change it until the card to create a new Posgres server in Azure is complete.  This change will be done as part of that one.